README.md

```diff
@@ -1000,7 +1000,7 @@ if (typeof console === "undefined") {
 
 Understand the difference between `escape()` `encodeURI()` `encodeURIComponent()` [here](http://stackoverflow.com/a/3608791/1748884).
 
-It worth mentioning that using `encodeURI()` and `encodeURIComponent()` has exactly 11 characters difference.
+It worth mentioning that using `encodeURI()` and `encodeURIComponent()` has exactly 11 characters different.
```

Comment:
"has exactly 11 characters difference" is the right wording. The same wording can also be phrased as "has exactly a difference of 11 characters". But we cannot say: "has exactly a ***different*** of 11 characters".

Reply:
I'm not sure about that being correct.
but hey his is you guy's work, have it the way you want. Your choice.
I'm just trying to help.

Reply:
ok, as you want.